schema-registry/serdes/src/main/java/com/hortonworks/registries/schemaregistry/serdes/avro/AbstractAvroSnapshotSerializer.java

```diff
@@ -83,5 +111,20 @@ protected String getSchemaText(Object input) {
         Schema schema = AvroUtils.computeSchema(input);
         return schema.toString();
     }
+    
+    
+
+    protected void serializeSchemaVersion(OutputStream os, SchemaIdVersion schemaIdVersion) throws SerDesException {
+        serDesProtocolHandler.handleSchemaVersionSerialization(os, schemaIdVersion);
+    }
+
+    protected void serializePayload(OutputStream os, Object object) throws SerDesException {
+        Object value = object instanceof NonRecordContainer ? ((NonRecordContainer) object).getValue() : object;
+        serDesProtocolHandler.handlePayloadSerialization(os, object);
```

Comment:
correct, i was adding it ready for the kafka connect converter work. Until we add that it doesn't get invoked,